.zenodo.json

```diff
@@ -162,6 +162,9 @@
     {
       "name": "Nguyen, Ly"
     },
+    {
+      "name": "Reddigari, Samir"
```

Comment:
@reddigari Let me know if you have another name you prefer for publications, your affiliation, and your [ORCID](https://orcid.org/).

Reply:
Affiliation: BrainSpec, Boston, MA
ORCID: 0000-0003-1472-5881